src/App.js

```diff
@@ -2,6 +2,8 @@ import React from "react"
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom"
 import login from "./pages/login"
 import signup from "./pages/signup"
+import EditAccount from './pages/EditAccount'
```

Comment:
This is more of a pedantic issue, but react uses lowercase filenames: https://stackoverflow.com/questions/43973199/file-naming-conventions-in-reactjs#:~:text=File%20names%20must%20be%20all,Filenames'%20extension%20must%20be%20.